_posts/2020-01-23-operating-system-and-browser-accessibility-display-modes.md

```diff
@@ -0,0 +1,373 @@
+---
+layout: post
+title: "Operating System and Browser Accessibility Display Modes"
+description: "Accessibility display modes are are pre-defined display configurations you can tell your device to use to read and take action on content."
+author: eric_bailey
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://www.kooslooijesteijn.net/blog/dark-mode-design-considerations
+    title: "Dark mode design con­sider­ations"
+    source: Koos Looijesteijn
+  - url: https://adrianroselli.com/2017/11/os-high-contrast-versus-inverted-colors.html
+    title: "OS: High Contrast versus Inverted Colors"
+    source: Adrian Roselli
+  - url: https://valhead.com/2017/06/23/reduced-motion-query/
+    title: "The reduced motion query at a glance"
+    source: Val Head
+  - url: https://www.gwhitworth.com/blog/2017/04/how-to-use-ms-high-contrast/
+    title: "How to use -ms-high-contrast"
+    source: Greg Whitworth
+---
+
+Most computers, laptops, tablets, smartphones, and web browsers have specialized tools to help people read and take action on the content they display. Some of these tools are display modes, which are pre-defined display configurations you can tell your device to use.
+
+Five such modes are Dark Mode, Increased Contrast Mode, Inverted Colors Mode, Reduced Motion Mode, and High Contrast Mode. Following are explanations of what each of these modes is, who can benefit from it, how to enable it on your device or browser (if supported), and how to support it in code.
+
+## Dark Mode
+
+The visuals that make up most Operating Systems (<abbr>OS</abbr>es) traditionally use lighter colors like white and light grays. Dark Mode is a toggle that instructs these visuals to use darker colors, typically black and dark grays.
+
+It is helpful for:
+
+- People who want to reduce eye strain in a low or no light environment,
+- and/or who have certain [low vision](https://webaim.org/articles/visual/lowvision) or photosensitive conditions.
+
+An example of one of these conditions could be checking your phone in bed with the lights off.
```

Comment:
lights off isn't really a condition, more a low/no light environment.

Reply:
I'd chalk it up to situational? Trying to subtly introduce the idea that it doesn't need to be a medical condition to benefit from Dark Mode support.

Reply:
yeh.  i think it's a good example.  it's just the use of the term 'condition'.

since the previous sentence mentions "photosensitive conditions", the reader's going to be thinking of this term in a medical sense.

Reply:
Ah, I getcha. I'll rephrase to remove the word "condition."